Library/Formula/kafka.rb

```diff
@@ -14,7 +14,7 @@ class Kafka < Formula
 
   depends_on "gradle"
   depends_on "zookeeper"
-  depends_on :java => "1.7+"
+  depends_on :java => "1.7"
```

Comment:
In that case, I would suggest to stick with `1.7+` and add a `pour_bottle?` method to prevent bottling on `1.7` only system.